master/internal/db/postgres_experiments.go

```diff
@@ -928,6 +928,26 @@ WHERE t.id = $1`, &experiment, id); err != nil {
 	return &experiment, nil
 }
 
+// ExperimentWithoutConfigByTaskID looks up an experiment by a given taskID, returning an error
+// if none exists. It loads the experiment without its configuration, for callers that do not need
+// it, or can't handle backwards incompatible changes.
+func (db *PgDB) ExperimentWithoutConfigByTaskID(taskID model.TaskID) (*model.Experiment, error) {
```

Comment:
maybe use `db.Bun().NewRaw` instead? still a raw query but no new growth for `PgDB`.